NOTES.md

```diff
@@ -1,3 +1,132 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-12 20:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+This is a beta release for RethinkDB 2.1. **It is not for production use and has known bugs.** Please do not use this version for production data.
+
+We are looking forward to your bug reports on GitHub (http://github.com/rethinkdb/rethinkdb/issues/new) or on our mailing list (https://groups.google.com/forum/#!forum/rethinkdb).
+
+Release highlight:
+
+* Automatic failover
+* More flexible administration for servers and tables
+* Advanced recovery features
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdbblog.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. However the final 2.1.0
+version will include that feature for versions of RethinkDB down to
+1.14.0. Upgrading directly from RethinkDB 1.13 will require manually
```

Comment:
I would break out the note about 1.13 into a separate line and say:

> If you're upgrading from RethinkDB 1.13 or earlier, you will need to manually upgrade using `rethinkdb dump`.